docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md

```diff
@@ -62,379 +111,293 @@ String database = "public";
 // By default, GreptimeDB listens on port 4001 using the gRPC protocol.
 // We can provide multiple endpoints that point to the same GreptimeDB cluster.
 // The client will make calls to these endpoints based on a load balancing strategy.
+// The client performs regular health checks and automatically routes requests to healthy nodes,
+// providing fault tolerance and improved reliability for your application.
 String[] endpoints = {"127.0.0.1:4001"};
 // Sets authentication information.
 AuthInfo authInfo = new AuthInfo("username", "password");
 GreptimeOptions opts = GreptimeOptions.newBuilder(endpoints, database)
-        // If the database does not require authentication, we can use AuthInfo.noAuthorization() as the parameter.
+        // If the database does not require authentication, we can use `AuthInfo.noAuthorization()` as the parameter.
         .authInfo(authInfo)
         // Enable secure connection if your server is secured by TLS
         //.tlsOptions(new TlsOptions())
         // A good start ^_^
         .build();
 
+// Initialize the client
 GreptimeDB client = GreptimeDB.create(opts);
 ```
 
```

Comment:
I think we shoud add some notes for the client instance: it's thread-safe, and recommend to reuse it as the global singleton instance etc. @fengjiachun